### PR TITLE
feat: add derivation path modal for trezor wallet

### PR DESCRIPTION
**Pull request by RyukTheCoder**
# Summary

Added derivation path modal for Trezor wallet.

Fixes # 1633

# How did you test this change?

Tested by connecting to Trezor wallet on different paths on Ethereum chain (EVM namespace) and signing transactions.

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

File: wallets/provider-trezor/src/index.ts [hunk 1]
```diff
@@ -29,18 +34,44 @@ export const init = (environments: Environments) => {
...
+      if (!isTrezorInitialized) {
+        await TrezorConnect.init({
+          lazyLoad: true, // this param will prevent iframe injection until TrezorConnect.method will be called
+          manifest: trezorManifest,
+        });
+
+        isTrezorInitialized = true;
+      }
+
+      const accounts = await getEthereumAccounts(getDerivationPath());
```
Comment (mikasackermn) [inline wallets/provider-trezor/src/index.ts:61]: Handle getDerivationPath() in the function itself. Its structure should be better same as Ledger, no need to offer it as a param.

**Review by mikasackermn — COMMENTED** [review]
Tnx, I just added a comments

File: wallets/provider-trezor/src/index.ts (hunk 1 above)
Comment (RyukTheCoder) [reply wallets/provider-trezor/src/index.ts]: Tnx Fixed

**Review by mikasackermn — APPROVED** [review]
LGTM

**Review by yeager-eren — APPROVED** [review]
lgtm. as we talked, just add a comment on top `getTrezorNormalizedDerivationPath` to desrcibe the use case.

**Pull request merged**